core/src/fi/jarno/sumblocks/resurrection/Actors/Block.java

```diff
@@ -31,11 +30,11 @@ public Block(float x, float y, float width, float height, int column, int row, i
         super();
         setBounds(x, y, width, height);
         setOrigin(width / 2, height / 2);
+        setScale(0);
```

Comment:
Scale should be set to 0 by hand if needed. Now this happens under the hood and might be confusing.